app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class variable. This means it is shared across all instances of `Person`. If this is intended to store all created `Person` instances, it should be handled carefully to avoid unintended overwrites or errors.